v1.2-client.yml

```diff
@@ -0,0 +1,1797 @@
+      - type: object
+        properties:
+          data:
+            $ref: TeacherObject
+          previous_attributes:
+            additionalProperties:
+              type: object
+
+  teachers.created:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: TeacherObject 
+
+  teachers.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: TeacherObject
+
+  sections.updated:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SectionObject
+          previous_attributes:
+            additionalProperties:
+              type: object
+
+  sections.created:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SectionObject 
+
+  sections.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SectionObject
+
+  schools.updated:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolObject
+          previous_attributes:
+            additionalProperties:
+              type: object
+
+  schools.created:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolObject 
+
+  schools.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolObject
+
+  schooladmins.updated:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolAdminObject
+          previous_attributes:
+            additionalProperties:
+              type: object
+
+  schooladmins.created:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolAdminObject 
+
+  schooladmins.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: SchoolAdminObject
+
+  studentcontacts.updated:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: StudentContactObject
+          previous_attributes:
+            additionalProperties:
+              type: object
+
+  studentcontacts.created:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: StudentContactObject 
+
+  studentcontacts.deleted:
+    allOf:
+      - $ref: "#/definitions/Event"
+      - type: object
+        properties:
+          data:
+            $ref: StudentContactObject
+
+  districts.updated:
```

Comment:
they're used by the java code to implement polymorphism. check out https://github.com/Clever/clever-java/blob/a0105a0205baeb1f91a6350ef62448f2774efa96/override/JSON.java#L101-L136